src/lib/hooks/useEmail.ts

```diff
@@ -24,7 +25,7 @@ export function useEmail() {
   const needsEmailVerification =
     !isSelfHost &&
     checkEmailConfirmed &&
-    !!currentAccount?.emailConfirmed &&
```

Comment:
Oops! This is fixed in the web production build, gotta merge it into main though.